source/agora/node/Ledger.d

```diff
@@ -163,15 +167,24 @@ public class Ledger
 
         Params:
             block = the block to add
+            active_enrollments = the number of enrollments that do not expire
+                at the block height.
+                if this value is size_t.max, it is calculated from
+                `ValidatorSet`.
 
         Returns:
             true if the block was accepted
 
     ***************************************************************************/
 
-    public bool acceptBlock (const ref Block block) @safe
+    public bool acceptBlock (const ref Block block,
+        size_t active_enrollments = size_t.max) @safe
```

Comment:
Why does this need to be here? It looks like a workaround for something. We should just get the validator count from the enrollment manager, and that's it. No `active_enrollments` parameter in `acceptBlock` and also `onExternalized` should not have it either.

Reply:
In fact `validateBlock` should internally call `getValidatorCount()`

Reply:
I found in the unittest that there was a place to call `acceptBlock` directly.
I did this for the unittest.

https://github.com/bpfkorea/agora/blob/dffeab65a57dafae6c267f7d673b957a42f66c62/source/agora/node/Ledger.d#L645

Reply:
Ah! I know what you want.

Reply:
That's a good idea, but we're going to have to put enrollment in the previous test, so it'll take some time.

Reply:
I looked back and there were quite a few things that needed to be fixed.
I also fixed the code related to freezing period.
I also fixed the test code of agora.node.Ledger.